src/components/views/groups/GroupTile.js

```diff
@@ -81,7 +82,7 @@ const GroupTile = React.createClass({
         ) : null;
         // XXX: Use onMouseDown as a workaround for https://github.com/atlassian/react-beautiful-dnd/issues/273
         // instead of onClick. Otherwise we experience https://github.com/vector-im/riot-web/issues/6156
-        return <AccessibleButton className="mx_GroupTile" onMouseDown={this.onMouseDown}>
+        return <AccessibleButton className="mx_GroupTile" onMouseDown={this.onMouseDown} onClick={nop}>
```

Comment:
an inline arrow function also works (and avoids a `nop` function): `onClick={() => {}}`

Reply:
will the linter not complain?

Reply:
![image](https://user-images.githubusercontent.com/2403652/41483838-901a50fa-70d2-11e8-8c5f-7e39d4dc12ed.png)
YUUUUP

Reply:
Yep, creating anonymous functions in the render method is nonideal since it will break React's DOM diffing algorithm: better to define the function once and pass it in.